templates/default/logstash_jvm.options.erb

```diff
@@ -0,0 +1,79 @@
+## JVM configuration
+
+# Xms represents the initial size of total heap space
+# Xmx represents the maximum size of total heap space
+
+-Xms1g
+-Xmx1g
```

Comment:
Is this memory option something we would need to be able to override from a cluster definition?

Reply:
So far we have been using the default `jvm.options` file that comes with the logstash packaging. Since this PR is quite high priority to get in, I wanted to avoid setting up tons of attributes in the same PR.